raiden_contracts/contract_manager.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, path: Union[Path, Dict[str, Path]]) -> None:
                     precompiled_content = json.load(precompiled_file)
             except (JSONDecodeError, UnicodeDecodeError) as ex:
                 raise ContractManagerLoadError(
-                    f"Can't load precompiled smart contracts: {ex}",
+                    f'Can\'t load precompiled smart contracts: {ex}',
```

Comment:
Can't we use the same coding standard on both projects?

[ref under `Usage of single and double quotes`](https://github.com/raiden-network/raiden/blob/master/CONTRIBUTING.md#coding-style)

Reply:
I remembered `"Augusto's computer is awesome"` example.

I didn't find a way to configure `pylint-quotes` to allow that.

Reply:
Now I see
```
Additionally the string-quote can be configured to avoid escaping: by default it enforces one type but if using the other type would avoid some escaping then it enforces the other one. To use those smart types the config is 'single-avoid-escape', and 'double-avoid-escape'.
```

So I'll try this in this PR.   The escapes should disappear.